rtc-service/requests.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "errors"
+
+type CreateRoomRequest struct {
+	// The room name
+	Name string `json:"name"`
+}
+
+// Validates if the correct parameters are passed to the request.
+func (request *CreateRoomRequest) Validate() (isValid bool, err error) {
```

Comment:
This might be useful for validation: https://gin-gonic.com/docs/examples/custom-validators/